source/patterns/@aws-solutions-constructs/aws-lambda-elasticsearch-kibana/test/integ.domain-arguments.ts

```diff
@@ -27,7 +28,7 @@ const lambdaProps: lambda.FunctionProps = {
 };
 
 const esDomain = 'domain-' + Aws.ACCOUNT_ID;
-const cognitoDomain = 'domainargumentsdomain';
+const cognitoDomain = 'domainargumentsdomain1';
```

Comment:
Why? And why "1"? If it must change, is there a better name that clarifies the difference?

Reply:
It was failing to deploy in my local envrionment even though the stack was deleted, Cognito service was complaining about the existing domain, I assume some issue with the backend cleanup in Cognito service, I will revisit this change one more time to see if it's giving the same error still

Reply:
Renamed domain name to `mydomainfortesting`